spec/index.html

```diff
@@ -293,7 +293,22 @@ <h2>Typographical conventions</h2>
   </section>
 </section>
 
-<section id="conformance"></section>
+<section id="conformance">
+  <p>A conforming processor is a system which can generate
+    the <a>canonical n-quads form</a> of an <a>input dataset</a>
+    consistent with the algorithms defined in this specification.</p>
+
+  <p>The algorithms in this specification should be considered normative,
```

Comment:
```suggestion
  <p>The algorithms in this specification are normative,
```

(A feature is either normative or not; it is not "considered"...:-)